job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -124,6 +124,7 @@ String kogitoNightlyStage(String kogitoVersion, String kogitoBranch, String opta
                 build job: 'kogito.nightly/${kogitoBranch}', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/repositories/rhba-${getNexusFromVersion(rhbaVersion)}-nightly-with-upstream'],
                         [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: '${rhbaVersionPrefix}'],
+                        [\$class: 'StringParameterValue', name: 'RHBA_RELEASE_VERSION', value: '${getNexusFromVersion(rhbaVersion)}'],
```

Comment:
Based on that, it will use `kogito-nightly-version` `SETTINGS_XML_ID`:
https://github.com/kiegroup/kogito-pipelines/blob/d501e310d7b6e3fac919a2e764ab9e27d384f39d/.ci/jenkins/Jenkinsfile.prod.nightly#L52

Can you look that the `version` part means it will actually use a group for 7.13?

Reply:
we were discussing exactly about that and @winklerm is actually right. See
```
$ diff managed-files_kogito-nightly-main managed-files_kogito-nightly-version 
5,7c5,7
<       <id>rhba-main-nightly-with-upstream</id>
<       <name>RHBA Main Nightly</name>
<       <url>https://^${env.LOCAL_NEXUS_IP}:8443/nexus/content/groups/rhba-main-nightly-with-upstream</url>
---
>       <id>rhba-kie-version-nightly-with-upstream</id>
>       <name>RHBA Kie Version Nightly</name>
>       <url>https://^${env.LOCAL_NEXUS_IP}:8443/nexus/content/groups/rhba-^${env.RHBA_RELEASE_VERSION}-nightly-with-upstream</url>
162c162
< </settings>
\ No newline at end of file
---
> </settings>
```
see https://issues.redhat.com/browse/RHPAM-4249

Reply:
Yes, I have checked, it substitutes the value of `RHBA_RELEASE_VERSION` so it should work for both `main` and `7.*` version.

Reply:
Nice, no further objections then! 😆